### ISSUE TITLE
Out of date `bs-platform` dependency

### ISSUE THREAD

**Issue by canadaduane** [OvermindDL1/bucklescript-tea#64]
I think the `bs-platform` dependency needs to be bumped, if it's supported:

```
  "devDependencies": {
    "bs-platform": "^2.1.0",
    ...
  }
```

causes the warning:

```
warning "bucklescript-tea@0.6.11" has incorrect peer dependency "bs-platform@^1.7.5"
```

**Comment by OvermindDL1** [comment]
Does it work with bs-platform 2.1.0?  I know it had some API changes and I've been intending to test that (hence why I'm still using 1.7.5 here) to make sure it still works, have you tested?

**Comment by canadaduane** [comment]
I have not yet. The other issue (error due to conflicting bs-platforms
installed on my system) was getting in the way of trying anything. That
issue is resolved, so I can try new things now.

On Dec 11, 2017 10:54 AM, "OvermindDL1" <notifications@github.com> wrote:

Does it work with bs-platform 2.1.0? I know it had some API changes and
I've been intending to test that (hence why I'm still using 1.7.5 here) to
make sure it still works, have you tested?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/OvermindDL1/bucklescript-tea/issues/64#issuecomment-350821143>,
or mute the thread

**Comment by tcoopman** [comment]
https://github.com/tcoopman/bucklescript-tea-starter-kit works with 2.1

**Comment by OvermindDL1** [comment]
@tcoopman Ah so it works fine, I can bump the peer version to 2.1 without issue then?  :-)

**Comment by tcoopman** [comment]
I haven't tested all the features of course. But the basis works fine. I
would say to bump it.

**Comment by OvermindDL1** [comment]
Published as 0.7.0.  :-)
